Apps/Playground/Scripts/experience.ts

```diff
@@ -390,15 +410,16 @@ CreateBoxAsync(scene).then(function () {
                     if (hololens) {
                         // Pass through, head mounted displays (HoloLens 2) require autoClear and a black clear color
                         xrSessionManager.scene.autoClear = true;
-                        xrSessionManager.scene.clearColor = new BABYLON.Color4(0, 0, 0, 0);
+                        xrSessionManager.scene.clearColor = new Color4(0, 0, 0, 0);
                     }
                 });
             });
         }, 5000);
     }
 
     if (text) {
-        const Writer = BABYLON.MeshWriter(scene, { scale: 1.0, defaultFont: "Arial" });
+        // @ts-ignore
```

Comment:
What's this for?

Reply:
Maybe put a comment after `// @ts-ignore` to explain why.

Reply:
fixed